README.md

```diff
@@ -25,7 +25,7 @@ Furthermore, Kafka requires ZooKeeper for coordination, and this cookbook does
 not install or manage ZooKeeper to any extent.
 A general recommendation is to not run Kafka and ZooKeeper on the same hardware.
 
-Ruby 1.9.3+ and Chef 11.6.0+.
+Ruby 2.3.1+ and Chef 12.1.0+.
```

Comment:
I think that we should just specify the Chef version, as Chef bundles Ruby and we don't have any control over what version they include. For development purposes there's `.ruby-version`.